payment_payfip/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,10 @@
+To use this module you'll need to get your **Nº Client Payfip**
+ 
+You can do so by following the steps described on 
+`the official website <https://www.payfip.gouv.fr/>`_.
```

Comment:
@ivantodorovich 
Maybe that page can be used too for the documentation : 
https://www.collectivites-locales.gouv.fr/finances-locales/offre-de-paiement-en-ligne-payfip